lang/java/reef-tang/tang/src/test/java/org/apache/reef/tang/formats/TestClassHierarchyRoundTrip.java

```diff
@@ -16,7 +16,7 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-package org.apache.reef.tang.implementation.protobuf;
+package org.apache.reef.tang.formats;
```

Comment:
This testing should not be moved to Format. Format is mainly for Confgiuration/Configuration module related stuff.